app/views/users/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<% require 'date' %>
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Username:</strong>
+  <%= @user.username %>
+</p>
+
+<h2> <%= @user.username %>'s events </h2>
+<ul id="events">
+  <% @user.created_events.each do |event| %>
+    <li>
+      <%= link_to event.description, event_path(event.id) %>
+    </li>
+  <% end %>
+</ul>
+
+<h2> Upcoming Events </h2>
+<ul id="events">
+  <% @user.attended_events.each do |event| %>
+    <% if event.event_date >= Date.current %>
+      <li>
+        <%= link_to event.description, event_path(event.id) %>
+      </li>
+    <% end %>
+
+  <% end %>
+</ul>
+<h2> Past Events </h2>
+<ul id="events">
+  <% @user.attended_events.each do |event| %>
+    <% if event.event_date < Date.current %>
+      <li>
+        <%= link_to event.description, event_path(event.id) %>
+      </li>
+    <% end %>
```

Comment:
1. Remove logic from views. Move them into view helpers
2. Instead of comparing against the current date, take advantage of the Event scope by doing to something link this
```rb
# Into the controllers UsersController#show
@past_events = @user.attended_events.past

# Into the view
<% @past_events.each do |event| %>
...
<% end %>
```